src/DevTools/MockRelayRenderer.tsx

```diff
@@ -115,7 +118,7 @@ export const MockRelayRenderer = ({
   })
 
   return (
-    <ContextProvider relayEnvironment={environment}>
+    <ContextProvider {...contextProps} relayEnvironment={environment}>
```

Comment:
I think it would make more sense for the `ContextProvider` in this file to include props of an existing previously applied context rather than overloading this component with the ability to inject context props.

Reply:
E.g.

```tsx
return (
  <ContextConsumer>
    {contextProps => (
      <ContextProvider {...contextProps} relayEnvironment={environment}>
        …
      </ContextProvider>
    )}
  </ContextConsumer>
)
```

Usage could then look like:

```tsx
renderRelayTree({
  …,
  wrapper: renderer => <ContextProvider mediator={…}>{renderer}</ContextProvider }>
})
```

Reply:
Updated!